sno/working_copy/base.py

```diff
@@ -53,25 +57,26 @@ def STATE_TABLE(self):
         return self._sno_table(self.STATE_NAME)
 
     @classmethod
-    def get(cls, repo, init_if_missing=False):
+    def get(cls, repo, allow_uncreated=False, allow_invalid_state=False):
         """
         Get the working copy associaated with this sno repo.
         If no working copy is present, returns None -
-        unless init_if_missing is returned, in which case it returns a WorkingCopy object that is not "created".
+        unless allow_uncreated is returned, in which case it returns a WorkingCopy object that is not "created".
         The type of this WorkingCopy object depends on the repo config - it represents a particular type of WorkingCopy
         at a particular location, which nevertheless doesn't exist yet. It is up to the caller to create it by calling
         `create()` and to populate it by calling eg `write_full()`.
 
         """
-        if init_if_missing:
-            cls.ensure_config_exists(repo)
-
         repo_cfg = repo.config
-        path_key = SNO_WORKINGCOPY_PATH
+        path_key = cls.SNO_WORKINGCOPY_PATH
         if path_key not in repo_cfg:
             return None
```

Comment:
>          If no working copy is present, returns None -
>          unless allow_uncreated is returned, in which case it returns a WorkingCopy object that is not "created".

seems to conflict with this. docstring not quite accurate?

Reply:
Fixed up the doc.